src/parser/metrics/coupling/Coupling.ts

```diff
@@ -95,6 +99,18 @@ export class Coupling implements CouplingMetric {
 
         couplingMetrics = this.calculateCouplingMetrics(relationships);
 
+        if (this.config.needsPrintPathFormatting()) {
+            for (const relationship of relationships) {
+                relationship.fromSource = formatPrintPath(relationship.fromSource, this.config);
+                relationship.toSource = formatPrintPath(relationship.toSource, this.config);
+            }
+            const relCouplingMetrics = new Map();
+            for (const [absolutePath, metricValues] of couplingMetrics) {
+                relCouplingMetrics.set(formatPrintPath(absolutePath, this.config), metricValues);
+            }
+            couplingMetrics = relCouplingMetrics;
+        }
```

Comment:
Let's extract this into a separate method.

Reply:
Done